src/validation.h

```diff
@@ -192,7 +192,7 @@ void PruneBlockFilesManual(int nManualPruneHeight);
 /** (try to) add transaction to memory pool
  * plTxnReplaced will be appended to with all transactions replaced from mempool
  * @param[out] fee_out optional argument to return tx fee to the caller **/
-bool AcceptToMemoryPool(CTxMemPool& pool, TxValidationState &state, const CTransactionRef &tx,
+bool AcceptToMemoryPool(CChainState& active_chainstate, CTxMemPool& pool, TxValidationState &state, const CTransactionRef &tx,
```

Comment:
In commit "validation: Pass in chainstate to ::AcceptToMemoryPool" (f75c7ca1cb46eb5824c3c5db666143f9a74c0eda)

This commit also doesn't compile, looks like there are calls in net_processing to update